README.md

```diff
@@ -0,0 +1,53 @@
+# Data Source Guide Example - Weather API
+
+A simple weather API built with FastAPI. This API provides random weather data based on latitude and longitude coordinates.
+
+## Requirements
+
+- [Pre-commit](https://pre-commit.com/#install)
+- [Python 3.11+](https://www.python.org/downloads/)
+- [Poetry](https://python-poetry.org/docs/#installation)
+
+To set up the `pre-commit` hooks, run `pre-commit install` in the repo. After it you can
+manually run `pre-commit` only for your changes or `pre-commit run --all-files` for all
+files.
+
+### Install Dependencies
+
+To install dependencies, first set up your environment with Poetry:
+
+```bash
+# Using Poetry
+poetry install
+
+### Run the App
```

Comment:
Code samples using triple backticks seem to be off a bit. Also we try to use more generic "shell" rather than "bash" for the examples.